Build failure: wivrn

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

1. Update from nixos/nixpkgs@3ce5398bca4f937a37cbb07c9d18cb2c37be43a8 (2025-10-15) to nixos/nixpkgs@534d5ee716a5f547efa4f3563d3592527e30329c (2025-11-14)
2. Enable cuda support for wivrn
3. Rebuild flake

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/313828007

### Relevant log output

```console
/build/source/server/encoder/ffmpeg/video_encoder_va.cpp: In constructor 'wivrn::video_encoder_va::video_encoder_va(wivrn::wivrn_vk_bundle&, wivrn::encoder_settings&, float, uint8_t)':
/build/source/server/encoder/ffmpeg/video_encoder_va.cpp:231:48: error: 'FF_PROFILE_H264_CONSTRAINED_BASELINE' was not declared in this scope; did you mean 'AV_PROFILE_H264_CONSTRAINED_BASELINE'?
  231 |                         encoder_ctx->profile = FF_PROFILE_H264_CONSTRAINED_BASELINE;
      |                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                AV_PROFILE_H264_CONSTRAINED_BASELINE
/build/source/server/encoder/ffmpeg/video_encoder_va.cpp:237:75: error: 'FF_PROFILE_HEVC_MAIN_10' was not declared in this scope; did you mean 'AV_PROFILE_HEVC_MAIN_10'?
  237 |                         encoder_ctx->profile = settings.bit_depth == 10 ? FF_PROFILE_HEVC_MAIN_10 : FF_PROFILE_HEVC_MAIN;
      |                                                                           ^~~~~~~~~~~~~~~~~~~~~~~
      |                                                                           AV_PROFILE_HEVC_MAIN_10
/build/source/server/encoder/ffmpeg/video_encoder_va.cpp:237:101: error: 'FF_PROFILE_HEVC_MAIN' was not declared in this scope; did you mean 'AV_PROFILE_HEVC_MAIN'?
  237 |                         encoder_ctx->profile = settings.bit_depth == 10 ? FF_PROFILE_HEVC_MAIN_10 : FF_PROFILE_HEVC_MAIN;
      |                                                                                                     ^~~~~~~~~~~~~~~~~~~~
      |                                                                                                     AV_PROFILE_HEVC_MAIN
/build/source/server/encoder/ffmpeg/video_encoder_va.cpp:240:48: error: 'FF_PROFILE_AV1_MAIN' was not declared in this scope; did you mean 'AV_PROFILE_AV1_MAIN'?
  240 |                         encoder_ctx->profile = FF_PROFILE_AV1_MAIN;
      |                                                ^~~~~~~~~~~~~~~~~~~
      |                                                AV_PROFILE_AV1_MAIN
make[2]: *** [server/CMakeFiles/wivrn-server.dir/build.make:583: server/CMakeFiles/wivrn-server.dir/encoder/ffmpeg/video_encoder_va.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
make[1]: *** [CMakeFiles/Makefile2:1675: server/CMakeFiles/wivrn-server.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.2, NixOS, 25.11 (Xantusia), 25.11.20251015.3ce5398`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.32.0`
 - nixpkgs: `/nix/store/2lay9g692wbzvgqsacqahf4n1l84v6lh-source`

### Notify maintainers

@ImSapphire @PassiveLemon

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/wi/wivrn/package.nix
+++ b/pkgs/by-name/wi/wivrn/package.nix
@@ -53,13 +53,13 @@
 }:
 stdenv.mkDerivation (finalAttrs: {
   pname = "wivrn";
-  version = "25.9";
+  version = "25.11.1";
 
   src = fetchFromGitHub {
     owner = "wivrn";
     repo = "wivrn";
     rev = "v${finalAttrs.version}";
-    hash = "sha256-XP0bpXgtira2QIlS0fNEteNP48WnEjBYFM1Xmt2sm5I=";
+    hash = "sha256-pEKMeRdI9UhdZ+NksRBcF7yPC7Ys2haE+B4PPGQ4beE=";
   };
 
   monado = applyPatches {
@@ -67,8 +67,8 @@ stdenv.mkDerivation (finalAttrs: {
       domain = "gitlab.freedesktop.org";
       owner = "monado";
       repo = "monado";
-      rev = "7a4018e2d89151e60e562fac79eba90ca7a328d8";
-      hash = "sha256-DPIvJb23bK7SDjZr9mK0Wt6Zbo3Ari3Ar8TtPe5QgKY=";
+      rev = "06e62fc7d9c5cbcbc43405bb86dfde3bf01ce043";
+      hash = "sha256-0ALB9eLY4NAUqNOYZMwpvYnLxVpHsQDJc1er8Txdezs=";
     };
 
     postPatch = ''
@@ -89,15 +89,6 @@ stdenv.mkDerivation (finalAttrs: {
       return 1
     fi
   '';
-
-  patches = [
-    # See https://github.com/WiVRn/WiVRn/pull/557
-    (fetchpatch {
-      name = "wivrn-fix-qt6.10-build.patch";
-      url = "https://github.com/WiVRn/WiVRn/commit/2204fdd39682cfc052556d58fdb9404dd8ecf63f.patch?full_index=1";
-      hash = "sha256-05MLfJNCznBt6eaggUfSk1jaNDB2/eou6CfexUkIHZE=";
-    })
-  ];
 
   nativeBuildInputs = [
     cmake

